src/index.html

```diff
@@ -21,7 +21,385 @@
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="card__review-text">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-numbers">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__link"
+          >
+            Buy
+          </a>
+        </div>
+
+        <!-- Card 7 -->
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            class="card__image"
+            alt="Picture of iMac 27 inch Retina"
+          />
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="card__review">
+            <div class="card__stars stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="card__review-text">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-numbers">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__link"
+          >
+            Buy
+          </a>
+        </div>
+
+        <!-- Card 8 -->
+        <div class="catalog__card card">
+          <img
+            src="./images/imac.jpeg"
+            class="card__image"
+            alt="Picture of iMac 27 inch Retina"
+          />
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="card__review">
+            <div class="card__stars stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="card__review-text">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-numbers">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__link"
+          >
+            Buy
+          </a>
+        </div>
```

Comment:
The card elements are repeated multiple times with the same content. Consider using a templating engine or JavaScript to dynamically generate these cards. This will make your code more maintainable and reduce redundancy.